### PR TITLE
Add images to master

### PR DESCRIPTION
**Pull request by casassg**
Images by @elena20ruiz

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -23,8 +23,10 @@
     <div class="wrapper">
         
         <img style="max-width: 90%" src="/assets/img/hackupc_header.png">
-        <h3 class="minecraftia subtitle">> 500 HACKERS<br>36 HOURS</h3>
-    
+        <div class="subtitle">
+              <img src="/assets/img/hackers.png">
```
Comment (ofpau) [inline dist/index.html]: We're back to "+" 500 instead of ">" :cry: 
But nevermind, it's fine

Comment (casassg) [inline dist/index.html]: @elena20ruiz can you change the image to >?

**Comment by casassg** [comment]
Changing this to only add subscribe?

**Comment by casassg** [comment]
Closing this to add in a separate pull request

**Pull request closed without merging**